plugins/chat/spec/system/separate_sidebar_mode_spec.rb

```diff
@@ -205,6 +205,25 @@
         expect(sidebar_component).to have_section_link(channel_2.name, active: true)
       end
     end
+
+    context "with subfolder" do
+      let!(:channel_browse_page) { PageObjects::Pages::ChatBrowse.new }
+
+      before do
+        set_subfolder "/discuss"
+        chat_page.prefers_full_page
+      end
+
+      it "has the expected behavior" do
+        visit("/discuss/about")
+
+        sidebar_component.switch_to_chat
```

Comment:
Not sure why yet but this test fails for me here, as switch_to_chat takes me to `/discuss/discuss/chat`.